client/src/components/Modal.jsx

```diff
@@ -1,28 +1,85 @@
 import React from 'react';
+import moment from 'moment';
+import _ from 'underscore';
+import Item from './Item.jsx';
 import Review from './Review.jsx';
 import ReviewContainer from './ReviewContainer.jsx';
-import styles from './css/modalStyles.css.js';
+import modalStyles from './css/modalStyles.css.js';
 
 const Modal = (props) => {
   if (!props.showModal) {
     return null;
   } else {
     return (
-      <div style={styles.modal} onClick={props.onClose}>
-        <section style={styles.modalMain}>
-          {/* <div style={styles.reviewContent}> */}
-          <img style={styles.image} src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
-          <ReviewContainer showPrice="true"/>
-          {/* </div> */}
-          {/* <div> */}
-          <button style={styles.button} onClick={props.onClose}>
-            <img style={styles.buttonImg}src="https://cdn0.iconfinder.com/data/icons/elasto-online-store/26/00-ELASTOFONT-STORE-READY_close-512.png"></img>
+      <div style={modalStyles.modal} onClick={props.onClose}>
+        <section style={modalStyles.modalMain}>
+          <img style={modalStyles.image} src={reviews[0].item_img}></img>
+          <div>
+            <div style={modalStyles.userInfo}>
+              <img style ={modalStyles.profileImg} src={reviews[0].user_img}></img>
+              <div style={modalStyles.username}>{reviews[0].user_name}</div>
+              <div style={modalStyles.date}>{moment(reviews[0].created_at).format('MMM DD, YYYY')}</div>
+            </div>
+            <div style={modalStyles.review}>
+              <div>
+                {_.times(reviews[0].stars, (n) =>{
+                  return (
+                    <img style={modalStyles.star} src="https://s3-us-west-1.amazonaws.com/anstyicons/icon-star-512.png"></img>
+                  );
+                })}
+              </div>
+              <div>{reviews[0].review}</div>
+            </div>
+            <div style={modalStyles.item}>
+              <Item showPrice="true" review={reviews[0]}/>
+            </div>
+          </div>
+          <button style={modalStyles.button} onClick={props.onClose}>
+            <img style={modalStyles.buttonImg}src="https://cdn0.iconfinder.com/data/icons/elasto-online-store/26/00-ELASTOFONT-STORE-READY_close-512.png"></img>
           </button>
-          {/* </div> */}
         </section>
       </div>
     );
   }
 };
 
+var reviews = [
```

Comment:
I know you haven't hooked up your backend, but a recommendation would be to declare all your dummy data in your top level in exactly the same form you can expect to hook it up from the database; this was how the HR sprints were done (such as re.castly and movie list).

If you do that, then you can independently design your modules, and once they're done you don't have to come back to them to refactor the code - the idea of preserving interfaces plays into this.  Once you hook up your backend, your interface to this module will need to be adjusted which will take time.

To exemplify this point, you use `reviews[0]` many times in your code here, which could have been passed as a prop from your top level container as `review={reviews[0]}, and then you don't have to come back later and refactor the code here (granted, it's not a huge amount of work, so this isn't the best illustration).

Reply:
thanks for the advice. I'll be more efficient with this in the future